README.md

```diff
@@ -117,7 +117,7 @@ See `python generate.py --help` for more options.
 
 ## Finetune the model
 
-We provide a simple training script in `finetune.py` that instruction-tunes a pretrained model on the [Alpaca](https://github.com/tatsu-lab/stanford_alpaca) dataset and also integrates [LoRA](https://arxiv.org/abs/2106.09685).
+We provide a simple training script in `finetune_lora.py` that instruction-tunes a pretrained model on the [Alpaca](https://github.com/tatsu-lab/stanford_alpaca) dataset and also integrates [LoRA](https://arxiv.org/abs/2106.09685).
```

Comment:
We need to feature `finetune_adapter.py` as well in the README (separate section)